src/spikeinterface/core/recording_tools.py

```diff
@@ -929,7 +929,9 @@ def get_rec_attributes(recording):
     return rec_attributes
 
 
-def do_recording_attributes_match(recording1, recording2_attributes) -> bool:
+def do_recording_attributes_match(
+    recording1: "BaseRecording", recording2_attributes: bool, check_is_filtered: bool = True, check_dtype: bool = True
+) -> Tuple[bool, str]:
```

Comment:
```suggestion
) -> tuple[bool, str]:
```

I think we want to keep to the builtins going forward no?

Reply:
done